L1Trigger/L1TTrackMatch/plugins/L1TkMuonProducer.cc

```diff
@@ -635,6 +677,12 @@ std::vector<L1TkMuMantraDF::muon_df> L1TkMuonProducer::product_to_muvec(const EM
   std::vector<L1TkMuMantraDF::muon_df> result(l1mus.size());
   for (uint imu = 0; imu < l1mus.size(); ++imu) {
     auto& mu = l1mus[imu];
+
+    // dropping the emtf tracks with certain quality...
+    int emtfQual = (mu.Mode() == 11 || mu.Mode() == 13 || mu.Mode() == 14 || mu.Mode() == 15);
```

Comment:
do these quality values have particular meanings? (useful to explain such meanings in a comment, or use named constants if appropriate)